scripts/assets/test_file_exemptions.textproto

```diff
@@ -6,6 +6,7 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/ActivityMo
 exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/ActivityScope.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/InjectableAppCompatActivity.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/route/Route.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/utility/lifecycle/LifecycleSafeTimerFactory.kt"
```

Comment:
Two things:
1. Please remove the old one (since the file was moved).
2. Please ensure this is in the correct place (per lexicographic ordering).

Reply:
done